packages/shell-api/src/database.ts

```diff
@@ -617,6 +618,21 @@ export default class Database extends ShellApiWithMongoClass {
     }
   }
 
+  @returnsPromise
+  @apiVersions([])
```

Comment:
Since this method was not mentioned in stable api [supported commands](https://www.mongodb.com/docs/manual/reference/stable-api-changelog/#std-label-stable-api-changelog), I skipped adding anything to this. Let me know if this is not the case.

Reply:
This method doesn’t do anything beyond `create` and `insert` in terms of command usage (I think), so it should be supported … I think you can wait and see how `test_apistrict` does with the e2e test for this one

Reply:
Perfect, thanks :)